src/pages/User/components/PullRequests/UserInfo/Message/getMessage.js

```diff
@@ -0,0 +1,44 @@
+import { TOTAL_PR_COUNT, TOTAL_OTHER_PR_COUNT } from '../../../../../../config';
+
+const messages = [
+  "It's not too late to start!",
+  'Off to a great start, keep going!',
+  'Keep going!',
+  'Keep it up!',
+  "Half way there, don't stop now!",
+  "Now, don't stop!",
```

Comment:
Remove this message as it is similar to the one above.

In place of `Now, don't stop!` use `You are right on track!`